xml/smtp_banners.xml

```diff
@@ -1647,4 +1648,93 @@ The system or service fingerprint with the highest certainty overwrites the othe
     <param pos="0" name="service.product" value="Unknown"/>
     <param pos="1" name="host.name"/>
   </fingerprint>
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+        <fingerprint pattern=”^(\w+.\w+.\w)+ Service not available$>
+        <description>match to Service not available error
+        <example service.version=”4.3.2”> 4.3.2 Service not available <example>
+        <param pos="1" name="service.version"/>
+        <param pos=”0” name=”service.vendor” value=”new”/>
+        <param pos=”0” name=”service.family” value=”version”/>
+        <param pos=”0” name=”service.product” value=”version”/>
+        <fingerprint>
+         <fingerprint pattern="^(.+.{4})-\w.+echosting+[^ ]+(cafe+.+.+.[a-z]+[a-z]+[a-z])+ ESMTP$">
+        <description>match to echosting or ESMTP
+        <example service.version="e0000" website.version="cafe24.com"> e0000-e.echosting.cafe24.com ESMTP <example>
```

Comment:
I think we've been trying to standardize on `foo.bar` as the hostname.  The hostname  is data that we capture as part of this process but, with a few exceptions, almost no Recog fingerprints will be tailored to the hosting provider.